public/plugin.ts

```diff
@@ -100,7 +100,12 @@ export class OpendistroSecurityPlugin
 
           return renderApp(coreStart, depsStart as AppPluginStartDependencies, params, config);
         },
-        category: DEFAULT_APP_CATEGORIES.management,
+        category: {
+          id: 'odfe',
+          label: 'Open Distro for Elasticsearch',
+          euiIconType: 'logoKibana',
+          order: 2000,
```

Comment:
Did we talk to other plugin team to decide the order?

Reply:
https://quip-amazon.com/Ur4TABAa3Ocd/Kibana-Side-Menu-for-ODFE-1120. UX designer has decide the order below. Security is the last one